app/sqlitehelper.py

```diff
@@ -0,0 +1,31 @@
+import sqlite3
+from sqlite3 import Connection, Error
+
+DEFAULT_DATABASE = "sqlitedatabase.db"
+
+class SQLiteHelper:
+    """Class to manage data stored in an SQLite database"""
+    def __init__(self, db=DEFAULT_DATABASE) -> None:
+        self.db = db
+        self.connection = self.create_connection()
+
+    def create_connection(self) -> Connection:
+        """Open and return Connection to the SQLite database."""
+        connection = None
```

Comment:
Returning null can result in lots of null pointer errors in the future. Python especially is not a null safe language, what that means is you can never know whether an object is actually null or not until you run into a NPE.  First and foremost, how should we handle connection = null? That would mean there is no database right? Should we throw an error and handle it elsewhere? What happens when there is no database already created?

If you really want to return null, is it possible to show that the return type is nullable anywhere?

Reply:
I just read the sqlite3 documentation, and it seems like the database will be "implicitly creating it if it does not exist".  If this is the case, maybe something like this might be better.

```
try:
    return sqlite3.connect(self.db)
except Error as e:
    print('error message')
```
Will python complain about the return type or no?

I think avoiding returning null values whenever possible will save us a lot of debugging in the future.

Reply:
Those are valid points. I have removed the null return and updated this method in the latest commit.